docs/integration/cassandra.md

```diff
@@ -4,48 +4,48 @@
-sstable size                |  cassandra.sstable.size                             |  gauge        |  long          |  bytes                              |  Disk space used by SSTables belonging to this table
-sstables count              |  cassandra.sstable.count                            |  gauge        |  long          |                                     |  Number of SSTables on disk for this table
-bloom false positives       |  cassandra.bloom.falsepositives                     |  counter      |  long          |                                     |  The number of Bloom filter false positives
-bloom false positive ratio  |  cassandra.bloom.falsepositives.ratio               |  gauge        |  double        |                                     |  The ratio of Bloom filter false positives to total checks
-bloom disk space            |  cassandra.bloom.disk.space                         |  gauge        |  long          |  bytes                              |  Disk space used by the Bloom filters
-sstables per read           |  cassandra.number.of.sstables.per.read              |  gauge        |  double        |                                     |  Number of SSTables are accessed during a read. Includes sstables that undergo bloom-filter checks, even if no data is read from the sstable
-local write request         |  cassandra.local.write.requests                     |  counter      |  long          |                                     |  Local writes update the table's memtable and appends to a commitlog
-local write latency total   |  cassandra.local.write.requests.latency.time        |  counter      |  long          |  ms                                 |
-avg local write latency     |  cassandra.local.write.latency.avg                  |  gauge        |  double        |  ms                                 |
-local read request          |  cassandra.local.read.requests                      |  counter      |  long          |                                     |  Local reads retrieve data from a table's memtable and any necessary SSTables on disk
-local read latency total    |  cassandra.local.read.requests.latency.time         |  counter      |  long          |  ms                                 |
-avg local read latency      |  cassandra.local.read.latency.avg                   |  gauge        |  double        |  ms                                 |
+Metric Name<br> Key *(Type)* *(Unit)*                                                                     |  Description
+----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+leaving nodes<br>**cassandra.cluster.nodes.leaving** <br>*(long gauge)*                                   |
```

Comment:
Should "Long" be lowercased?

Reply:
it was lower case when displayed numeric type in a separate column. now we have combined numeric type and metric type. e.g `Long gauge` `Double counter` is fine?

Reply:
For some reason screenshot is showing lowercase, but source has uppercase? I prefer lowercased type, but it is ok.